assignments/function-conversion.js

```diff
@@ -1,30 +1,37 @@
 // Take the commented ES5 syntax and convert it to ES6 arrow Syntax
 
-// let myFunction = function () {
+// let myFunction = function () { 
 // console.log("Function was invoked!");
 // };
 // myFunction();
 
+let myFunction = function1 => console.log("Function was invoked!"); 
+
 // let anotherFunction = function (param) {
 //   return param;
 // };
 // anotherFunction("Example");
 
+let anotherFunction = (param) => {param}; 
+
```

Comment:
To test your function and to make sure is logging the result you intended, make sure to invoke the function. `anotherFunction()`